x-pack/solutions/observability/plugins/apm/public/components/app/onboarding/serverless_instructions.ts

```diff
@@ -58,51 +70,55 @@ export function serverlessInstructions(
-      defaultMessage: 'APM Agents',
-    }),
-    instructionVariants: [
-      {
-        id: INSTRUCTION_VARIANT.NODE,
-        instructions: createNodeAgentInstructions(commonOptions),
-      },
-      {
-        id: INSTRUCTION_VARIANT.DJANGO,
-        instructions: createDjangoAgentInstructions(commonOptions),
-      },
-      {
-        id: INSTRUCTION_VARIANT.FLASK,
-        instructions: createFlaskAgentInstructions(commonOptions),
-      },
-      {
-        id: INSTRUCTION_VARIANT.RAILS,
-        instructions: createRailsAgentInstructions(commonOptions),
-      },
-      {
-        id: INSTRUCTION_VARIANT.RACK,
-        instructions: createRackAgentInstructions(commonOptions),
-      },
-      {
-        id: INSTRUCTION_VARIANT.GO,
-        instructions: createGoAgentInstructions(commonOptions),
-      },
-      {
-        id: INSTRUCTION_VARIANT.JAVA,
-        instructions: createJavaAgentInstructions(commonOptions),
-      },
-      {
-        id: INSTRUCTION_VARIANT.DOTNET,
-        instructions: createDotNetAgentInstructions(commonOptions),
-      },
-      {
-        id: INSTRUCTION_VARIANT.PHP,
-        instructions: createPhpAgentInstructions(commonOptions),
-      },
-      {
-        id: INSTRUCTION_VARIANT.OPEN_TELEMETRY,
-        instructions: createOpenTelemetryAgentInstructions(commonOptions),
-      },
-    ],
-  };
+  return [
+    {
+      title: DEFAULT_INSTRUCTION_TITLE,
+      id: INSTRUCTION_VARIANT.NODE,
+      instructions: createNodeAgentInstructions(commonOptions),
+    },
+    {
+      title: DEFAULT_INSTRUCTION_TITLE,
+      id: INSTRUCTION_VARIANT.DJANGO,
+      instructions: createDjangoAgentInstructions(commonOptions),
+    },
+    {
+      title: DEFAULT_INSTRUCTION_TITLE,
+      id: INSTRUCTION_VARIANT.FLASK,
+      instructions: createFlaskAgentInstructions(commonOptions),
+    },
+    {
+      title: DEFAULT_INSTRUCTION_TITLE,
+      id: INSTRUCTION_VARIANT.RAILS,
+      instructions: createRailsAgentInstructions(commonOptions),
+    },
+    {
+      title: DEFAULT_INSTRUCTION_TITLE,
+      id: INSTRUCTION_VARIANT.RACK,
+      instructions: createRackAgentInstructions(commonOptions),
+    },
+    {
+      title: DEFAULT_INSTRUCTION_TITLE,
+      id: INSTRUCTION_VARIANT.GO,
+      instructions: createGoAgentInstructions(commonOptions),
+    },
+    {
+      title: DEFAULT_INSTRUCTION_TITLE,
+      id: INSTRUCTION_VARIANT.JAVA,
+      instructions: createJavaAgentInstructions(commonOptions),
+    },
+    {
+      title: DEFAULT_INSTRUCTION_TITLE,
+      id: INSTRUCTION_VARIANT.DOTNET,
+      instructions: createDotNetAgentInstructions(commonOptions),
+    },
+    {
+      title: DEFAULT_INSTRUCTION_TITLE,
+      id: INSTRUCTION_VARIANT.PHP,
+      instructions: createPhpAgentInstructions(commonOptions),
+    },
+    {
+      id: INSTRUCTION_VARIANT.OPEN_TELEMETRY,
+      instructions: createOpenTelemetryAgentInstructions(commonOptions),
+    },
+  ];
```

Comment:
Here I moved the `title` on the individual item level instead of the global one because Open Telemetry flow should not have a title and this gives us flexibility to customize titles for other flow if needed.